{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/templates/base.html

```diff
@@ -92,16 +97,19 @@
 {% if cookiecutter.frontend_pipeline == 'None' %}
   {% raw %}
   <script defer src="{% static 'js/project.js' %}"></script>
+  <script defer src="{% static 'js/theme-toggler.js' %}"></script>
 {%- endraw %}
 {% elif cookiecutter.frontend_pipeline == 'Django Compressor' %}
 {% raw %}
 {% compress js %}
   <script defer src="{% static 'js/project.js' %}"></script>
+  <script defer src="{% static 'js/theme-toggler.js' %}"></script>
 {% endcompress %}
 {%- endraw %}
 {% elif cookiecutter.frontend_pipeline == 'Gulp' %}
 {% raw %}
 <script defer src="{% static 'js/project.min.js' %}"></script>
+<script defer src="{% static 'js/theme-toggler.js' %}"></script>
```

Comment:
Would be nice if `theme-toggler.js` could be imported by `project.js` when using Gulp or Webpack. They will take care of bundling it nicely, I think.